cves/CVE-2012-6617.yml

```diff
@@ -238,8 +236,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: There was only one file call ffserver.c and it was in the main portion of the repository.
```

Comment:
Maybe just re-comment that this specific subsystem is now depreciated. I think it's a cool fact

Reply:
That is an important piece of information that I left out. Thank you!

Reply:
Updated